src/protocol/webdriver-bidi.ts

```diff
@@ -592,20 +594,58 @@ export namespace BrowsingContext {
     message: string;
   };
 }
-export type NetworkCommand = Record<string, never>;
-export type NetworkResult = Record<string, never>;
+export type NetworkCommand =
+  | Network.AddIntercept
+  | Network.ContinueRequest
+  | Network.ContinueResponse
+  | Network.ContinueWithAuth
+  | Network.FailRequest
+  | Network.ProvideResponse
+  | Network.RemoveIntercept;
+export type NetworkResult = {};
```

Comment:
Can you manually change this to `Record<string,never>` this will cause issue with typings else . The latest CDDL parser has that change.

Reply:
Not applicable anymore, given:

```
export type NetworkResult = Network.AddInterceptResult;
```